migtests/scripts/functions.sh

```diff
@@ -1071,4 +1071,33 @@ EOF
 			exit 1
 			;;
 	esac
-}
\ No newline at end of file
+}
+
+normalize_and_export_vars() {
+    local test_suffix=$1
+
+    # Normalize TEST_NAME
+    export NORMALIZED_TEST_NAME="$(echo "$TEST_NAME" | tr '/-' '_')"
+
+    # Set EXPORT_DIR
+    export EXPORT_DIR=${EXPORT_DIR:-"${TEST_DIR}/${NORMALIZED_TEST_NAME}_${test_suffix}_export-dir"}
+    if [ -n "${SOURCE_DB_SSL_MODE}" ]; then
+        EXPORT_DIR="${EXPORT_DIR}_ssl"
+    fi
+
+    # Set database-specific variables
+    case "${SOURCE_DB_TYPE}" in
+        postgresql|mysql)
+            export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"${NORMALIZED_TEST_NAME}_${test_suffix}"}
+            ;;
+        oracle)
+            # Limit schema name to 10 characters for Oracle/Debezium due to 30 character limit
+            export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"${NORMALIZED_TEST_NAME:0:10}_${test_suffix}"}
```

Comment:
nit: you can do the truncate 0:10 also above when generating NORMALIZED_TEST_NAME.

Reply:
Keeping it specific to Oracle for now. The longer export-dir name brought out some issues. It'll be good to have that test case.